app/controllers/concerns/importable.rb

```diff
@@ -25,13 +25,17 @@ module Importable
   def import_csv
     if params[:file].present?
       data = File.read(params[:file].path, encoding: "BOM|UTF-8")
-      csv = CSV.parse(data, headers: true)
+      csv = CSV.parse(data, headers: true, skip_blanks: true)
```

Comment:
For consistence, do we actually want to add `skip_blanks: true` when we have a number of other places where we parse CSV data where we don't? It's easier to explain a rule on import to users that's true in all cases.

If we think we should allow+skip blanks on CSV import, can we leave that out here and do it in a separate PR updating every `CSV.parse` we're using?